src/tiling/tile_state.rs

```diff
@@ -117,6 +129,13 @@ impl<'a, T: Pixel> TileStateMut<'a, T> {
         })
         .collect(),
       rdo: RDOTracker::new(),
+      stripe_filter_integral_image: vec![0; STRIPE_FILTER_INTEGRAL_IMAGE_SIZE],
```

Comment:
Maybe it would be better to store and pass these in some sort of struct.

Reply:
That requires solving the hardest problem in computer science

Reply:
SelfGuidedRestorationScratch maybe

Reply:
Sure, or IntegralImageBuffer. There are only 4 so a struct isn't super critical.